Sources/Extensions/OAHudButton.h

```diff
@@ -0,0 +1,25 @@
+//
+//  OAHudButton.h
+//  OsmAnd Maps
+//
+//  Created by nnngrach on 13.07.2021.
+//  Copyright © 2021 OsmAnd. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface OAHudButton : UIButton
```

Comment:
In objective c and swift and extension has a different meaning to a subclass. Please do not put subclasses into a folder called Extensions.